squad/frontend/templates/squad/test_run.html

```diff
@@ -63,64 +63,84 @@
 
 
 {% if tests_status|length > 0 %}
-<h2>Tests</h2>
-<table class='table table-hover table-condensed'>
+    <h2>Tests</h2>
     {% for status in tests_status %}
-    <tr class='warning'>
-        <td>
-            <h4>{{status.suite.slug}}</h4>
-        </td>
-        <td style='vertical-align: middle'>
-            <div class="progress" style='margin-bottom: 0px'>
-                <div class="progress-bar progress-bar-success" style="width: {{status.pass_percentage}}%" title="Pass: {{status.pass_percentage|stringformat:"d"}}%">
-                    <span class="sr-only">Pass: {{status.pass_percentage}}% </span>
+        <a name="tests-{{status.suite}}"></a>
+        <h3>{{status.suite}}</h3>
+        {% with tests=status.tests_by_result %}
+            {% if tests.fail %}
+                {% for test in status.tests_by_result.fail %}
+                    <div class='row fail'>
+                        <div class='col-md-3 col-sm-3'>
+                            <i class='fa fa-times'></i>
+                            {{test.name}}
+                        </div>
+                        <div class='col-md-9 col-sm-9'>
```

Comment:
IMHO this should be hidden by default. Snippets can be long.

Reply:
fixed

![screen shot 2017-10-05 at 10 13 33](https://user-images.githubusercontent.com/30027/31229062-e5808f08-a9b5-11e7-865c-5d570f90474b.png)
![screen shot 2017-10-05 at 10 13 37](https://user-images.githubusercontent.com/30027/31229061-e57e74f2-a9b5-11e7-9c1d-d0e42c54e785.png)